### PR TITLE
[DOC] Change APM to service graph view

### PR DESCRIPTION
**Pull request by knylander-grafana**
**What this PR does**:
Renames the APM dashboard to the Service graph view to clarify what the view shows. 

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Checklist**
- [ ] Tests updated
- [X] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

**Review by zalegrala — APPROVED** [review]

**Review by zalegrala — APPROVED** [review]

File: docs/sources/tempo/metrics-generator/service-graph-view.md [hunk 1]
```diff
@@ -1,51 +1,51 @@
...
+aliases:
+- /docs/tempo/latest/metrics-generator/app-performance-mgmt
 weight: 200
 ---
 
-# Application performance management dashboard
+# Service graph view
 
-Application performance management (APM) uses data gathered from services, agents, systems, and microservices to visualize and identify areas of concern.
+The service graph view visualizes the span metrics (traces data for rates, error rates, and durations (RED)) and service graphs.
```
Comment (09jvilla) [inline docs/sources/tempo/metrics-generator/service-graph-view.md:11]:
```suggestion
Grafana's service graph view utilizes metrics generated by the metrics-generator to display span request rates, error rates, and durations, as well as service graphs.
```

File: docs/sources/tempo/metrics-generator/service-graph-view.md [hunk 2]
```diff
@@ -1,51 +1,51 @@
...
 
 You have to enable span metrics and service graph generation on the Grafana backend so metrics that are generated as traces are ingested.
 
-To use the APM dashboard, you need:
+To use the service graph view, you need:
 
 * Tempo or Grafana Cloud Traces with either 1) the metrics generator enabled and configured or 2) the Grafana Agent enabled and configured to send data to a Prometheus-compatible metrics store
 * [Services graphs]({{< relref "../metrics-generator/service_graphs/#how-to-run" >}}), which are enabled by default in Grafana
-* [APM table enabled](https://grafana.com/docs/grafana/latest/datasources/tempo/) within the service graph tab
+* [Span metrics table enabled](https://grafana.com/docs/grafana/latest/datasources/tempo/) within the service graph tab
```
Comment (09jvilla) [inline docs/sources/tempo/metrics-generator/service-graph-view.md:31]: Let's leave this for a separate PR, but I think this is going away now that we've eliminated the feature toggle

File: docs/sources/tempo/metrics-generator/service-graph-view.md [hunk 3]
```diff
@@ -1,51 +1,51 @@
...
 * [Span metrics]({{< relref "../metrics-generator/span_metrics/#how-to-run" >}}) enabled in your Tempo data source configuration
 
-The APM dashboard can be generated with the metrics-generator or Grafana Agent.
+The service graph view can be generated with the metrics-generator or Grafana Agent.
```
Comment (09jvilla) [inline docs/sources/tempo/metrics-generator/service-graph-view.md:34]:
```suggestion
The service graph view can be derived from metrics generated by either Tempo's metrics-generator or by the Grafana Agent.
```

File: docs/sources/tempo/getting-started/tempo-in-grafana.md [hunk 4]
```diff
@@ -74,15 +74,15 @@ for search of the backend datastore.
...
 
 
-## APM dashboard
+## Service graph view
 
-Grafana provides a built-in Application Performance Management (APM) dashboard available in Grafana Cloud and Grafana 9.1.
-The Grafana APM dashboard visualizes the span metrics (traces data for rates, error rates, and durations (RED)) and service graphs.
+Grafana provides a built-in service graph view available in Grafana Cloud and Grafana 9.1.
+The service graph view visualizes the span metrics (traces data for rates, error rates, and durations (RED)) and service graphs.
 Once the requirements are set up, this pre-configured dashboard is immediately available in **Explore > Service Graphs**.
```
Comment (09jvilla) [inline docs/sources/tempo/getting-started/tempo-in-grafana.md:81]:
```suggestion
Once the requirements are set up, this pre-configured view is immediately available in **Explore > Service Graphs**.
```

**Review by 09jvilla — CHANGES_REQUESTED** [review]
Left a few small suggestions.

File: docs/sources/tempo/metrics-generator/service-graph-view.md (hunk 2 above)
Comment (knylander-grafana) [reply docs/sources/tempo/metrics-generator/service-graph-view.md:31]: Actaully, I think that would come out entirely since APM table will be enabled by default, right?

Comment (knylander-grafana) [inline docs/sources/tempo/metrics-generator/service-graph-view.md:31]:
```suggestion
```

**Review by 09jvilla — APPROVED** [review]
LGTM minus 1 fix to my own proposed change :)

File: docs/sources/tempo/metrics-generator/service-graph-view.md [hunk 5]
```diff
@@ -1,51 +1,50 @@
...
+aliases:
+- /docs/tempo/latest/metrics-generator/app-performance-mgmt
 weight: 200
 ---
 
-# Application performance management dashboard
+# Service graph view
 
-Application performance management (APM) uses data gathered from services, agents, systems, and microservices to visualize and identify areas of concern.
+Grafana's service graph view utilizes metrics generated by the metrics-generator to display span request rates, error rates, and durations, as well as service graphs.
```
Comment (09jvilla) [inline docs/sources/tempo/metrics-generator/service-graph-view.md:11]:
```suggestion
Grafana's service graph view utilizes metrics generated by the metrics-generator (or Grafana Agent) to display span request rates, error rates, and durations, as well as service graphs.
```
